File: Math/Least Common Multiple/lcm.java
import java.util.Scanner;

public class Main {
  
  public static int gcd(int a, int b) {  // Greatest Common Divisor
    if(b == 0)
      return a;
    
    return gcd(b, a % b);
  }
  
  public static int lcm(int a, int b) { // Change type if needed
    return a / gcd(a, b) * b;
  }
  
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int a, b;
    
    a = sc.nextInt();
    b = sc.nextInt();
    System.out.println(lcm(a, b));
    
  }
}

File: Number Theory/Least Common Multiple/lcm.cpp
#include <iostream>

using namespace std;

typedef int T; // Change type if needed

T gcd(T a, T b) { // Greatest Common Divisor
  if(b == 0)
    return a;
  
  return gcd(b, a % b);
}

T lcm(T a, T b) {
  return a / gcd(a, b) * b;
}

int main() {
  T a, b;
  
  cin >> a >> b;
  cout << lcm(a, b) << endl;
  
  return 0;
}
